### PR TITLE
updated cluster loader image pull spec to use public registry again

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuation of https://github.com/openshift/openshift-docs/pull/23321 (rebased and solved merge conflicts)

This resolves https://bugzilla.redhat.com/show_bug.cgi?id=1817620 and https://github.com/openshift/openshift-docs/issues/23009

@JaredBurck FYI. Getting this moving along.

@RH-ematysek @cpmeadors Can you please provide QE review?

### CODE REVIEW

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Review by jtaleric — APPROVED** [review]
LGTM, thanks!

File: modules/running-cluster-loader.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ template builds and waits for them to complete:
 +
 ----
 $ sudo podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config:z -i \
-registry.svc.ci.openshift.org/ocp/4.5:tests /bin/bash -c 'export KUBECONFIG=/root/.kube/config && \
+quay.io/openshift/origin-tests:4.5 /bin/bash -c 'export KUBECONFIG=/root/.kube/config && \
 openshift-tests run-test "[sig-scalability][Feature:Performance] Load cluster \
 should populate the cluster [Slow][Serial] [Suite:openshift]"
```
Comment (cpmeadors) [inline modules/running-cluster-loader.adoc:23]: I think you are missing a closing single quote at the end of the command.

File: modules/running-cluster-loader.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ setting the environment variable for `VIPERCONFIG`:
 ----
 $ sudo podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config:z \
 -v ${LOCAL_CONFIG_FILE_PATH}:/root/configs/:z \
--i registry.svc.ci.openshift.org/ocp/4.5:tests \
+-i quay.io/openshift/origin-tests:4.5 \
 /bin/bash -c 'KUBECONFIG=/root/.kube/config VIPERCONFIG=/root/configs/test.yaml \
 openshift-tests run-test "[sig-scalability][Feature:Performance] Load cluster \
 should populate the cluster [Slow][Serial] [Suite:openshift]"
```
Comment (cpmeadors) [inline modules/running-cluster-loader.adoc:35]: Missing closing single quote here too.

**Review by cpmeadors — CHANGES_REQUESTED** [review]
Changes look good with the exception of the missing single quotes.  All commands execute correctly.  I verified that I removed credentialsfor registry.svc.ci.openshift.org. and was able to access quay.io without any special credentials.  Looks publicly accessible to me.

**Comment by ahardin-rh** [comment]
@cpmeadors Nice catch! Thank you for the review!

**Review by cpmeadors — COMMENTED** [review]
Closing single quotes are good.  Coping the command and pasting without any modifications works as expected now.

**Review by cpmeadors — COMMENTED** [review]
Approved

**Pull request merged**